source/code/utilities/chinese/chinese_translations.hcp

```diff
@@ -1080,10 +1080,65 @@ class
         Translation_Entry x;
         x.english = L"He did not know her when he was young.";
         x.pinyin = L"Tā xiǎo shíhou bù rènshi tā.";
-        x.chinese = L"他小时候不认识她";
+        x.chinese = L"他小时候不認識她";
         x.tags.emplace_back("phrase");
         entries.emplace_back(x);
     }
+    
+    {
+        Translation_Entry x;
+        x.english = L"Spring.";
+        x.pinyin = L"chūn tiān";
+        x.chinese = L"春天";
+        x.tags.emplace_back("season");
+        entries.emplace_back(x);
```

Comment:
I learned that tiān is day, so these should be easy to learn:
kind of like weekdays